src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainViewStats.tsx

```diff
@@ -0,0 +1,217 @@
+  exampleLinks?: Partial<urlObj>;
+  exampleLinkText?: string;
+}) => {
+  const { exampleLinks, exampleLinkText } = props;
+
+  const [showPointerBox, setShowPointerBox] = useState(false);
+  const anchorRef = useRef<HTMLSpanElement>(null);
+  const containerRef = useRef<HTMLSpanElement>(null);
+
+  if (!exampleLinks) {
+    return null;
+  }
+
+  return (
+    <div className={styles.exampleLink}>
+      <span ref={containerRef}>
+        <span
+          className={styles.exampleLinkText}
+          ref={anchorRef}
+          onClick={() => setShowPointerBox(!showPointerBox)}
+        >
+          {exampleLinkText}
+          {showPointerBox && (
+            <PointerBox
+              anchor={anchorRef.current as HTMLDivElement}
+              onOutsideClick={() => setShowPointerBox(false)}
+              position={Position.BOTTOM_RIGHT}
+              container={containerRef.current}
+              autoAdjust={true}
+              classNames={{
+                box: styles.pointerBox,
+                pointer: styles.pointerBoxPointer
+              }}
+            >
+              <ViewInApp links={exampleLinks} />
+            </PointerBox>
+          )}
+        </span>
+      </span>
+    </div>
+  );
+};
+
+const getCollapsedContent = (statsSection: StatsSection) => {
+  const { primaryStats, secondaryStats, section, exampleLinks } = statsSection;
+  const { title, exampleLinkText } = sectionGroupsMap[section];
+
+  return (
+    <div className={styles.collapsedContent}>
+      <span className={styles.title}>{title}</span>
+      {primaryStats && (
+        <div className={styles.primary}>
+          <span className={styles.primaryValue}>
+            {primaryStats.primaryValue}
+          </span>
+          <span className={styles.primaryUnit}>{primaryStats.primaryUnit}</span>
+        </div>
+      )}
+
+      {secondaryStats && (
+        <div className={styles.secondary}>
+          <span className={styles.secondaryValue}>
+            {secondaryStats.secondaryValue}
+          </span>
+          <span className={styles.secondaryUnit}>
+            {secondaryStats.secondaryUnit}
+          </span>
+        </div>
+      )}
+
+      <ExampleLinks
+        exampleLinks={exampleLinks}
+        exampleLinkText={exampleLinkText}
+      />
+    </div>
+  );
+};
+
+const getExpandedContent = (statsSection: StatsSection) => {
+  const { groups, exampleLinks, section } = statsSection;
+  const { exampleLinkText } = sectionGroupsMap[section];
+  return (
+    <div className={styles.expandedContent}>
+      {groups.map((group, group_index: number) => {
+        const { title, stats } = group;
+        const primaryStatsClassName = classNames(styles.primaryStats, {
+          [styles.primaryStatsNoLink]: exampleLinks
+            ? !Object.keys(exampleLinks).length
+            : true
+        });
+        const primaryOrSecondaryClassName =
+          group_index === 0 ? primaryStatsClassName : styles.secondaryStats;
```

Comment:
I tried to figure this out from the code and from the way the page looks, and couldn't. What's the difference between "primary stats" and "secondary stats"? Is it just the first stats group versus all others? Then shouldn't it be something like `statsGroup` vs `firstStatsGroup`? Or, for more convenience with SCSS, expressing it in the BEM style: `statsGroup` vs `statsGroup_first`?

Reply:
Updated to use `first-of-type` and removed the line above.